dbt/include/snowflake/macros/adapters.sql

```diff
@@ -155,14 +155,41 @@
     show terse objects in {{ schema_relation }}
   {%- endset -%}
 
+  {%- set fallback_sql -%}
+     select
+      created as "created_on",
+      table_name as "name",
+      case when table_type = 'BASE TABLE' then 'table'
+           when table_type = 'VIEW' then 'view'
+           when table_type = 'MATERIALIZED VIEW' then 'materializedview'
+           when table_type = 'EXTERNAL TABLE' then 'external'
+           else table_type
+      end as "kind",
+      table_catalog as "database_name",
+      table_schema as "schema_name"
+    from  information_schema.tables
+    where table_schema ilike '{{ schema_relation.schema.lower() }}'
```

Comment:
Would changing these filters from `ilike` to equality help with the performance issues?

Reply:
Hey @mikealfare thanks for reviewing this. I learned late yesterday that in fact we can paginate the `show terse objects` query on snowflake, so shifted direction there in working with @dbeatty10.